### PR TITLE
Add "Project Details" Meta-Box to Posts in admin

### PR DESCRIPTION
**Pull request by adamish717**
Tested and working but still may add more items dependent on what Windley wants on the individual project pages

### CODE REVIEW

**Comment by adamish717** [comment]
@Sinetheta Ready to be merged

**Comment by adamish717** [comment]
This has been updated to move things out of functions.php and in to folders more closely resembling plugins (except are still theme-specific, so they'll remain in an includes folder for this theme)

File: includes/custom-meta/_custom-meta.php [hunk 1]
```diff
@@ -0,0 +1,87 @@
+<?php
+
+// Style those forms a bit
+function admin_style() {
+  wp_enqueue_style('admin-styles', get_template_directory_uri().'/includes/custom-meta/meta-box-styles.css');
```
Comment (Sinetheta) [inline includes/custom-meta/_custom-meta.php:5]: I'd just add an `admin.scss` next to `main.scss` and import all admin styles into that.

**Comment by Sinetheta** [comment]
:+1:

**Pull request merged**